schemas/list-workers-response.yml

```diff
@@ -85,7 +85,12 @@ properties:
           title:  "URL"
           description: {$const: action-url-description}
           type:    string
-          format:  uri
+        method:
+          title:  "Method"
+          description: |
+            Method to indicate the desired action to be performed for a given resource.
+          type:   string
+          enum:   ["GET", "POST", "PUT", "DELETE", "PATCH"]
```

Comment:
I don't think GET should ever perform an action.

Reply:
This must have slipped in by accident. Fixing that right now!